src/app/configuration/configuration.component.ts

```diff
@@ -21,6 +25,30 @@ const LINKS: EntityLink[] = [
   templateUrl: './configuration.component.html',
   styleUrls: ['./configuration.component.scss'],
 })
-export class ConfigurationComponent {
+export class ConfigurationComponent implements OnDestroy {
   links: EntityLink[] = LINKS;
+
+  private subscription: Subscription[] = [];
+  public regionName: string | null;
+
+  constructor(private router: Router, private sendResourceService: SendResourceService) {
+    this.subscription.push(
+      this.sendResourceService.getResource().subscribe((response: any) => {
+        if (response.fromComponent === 'line-listing-component') {
```

Comment:
We could do this like it is in other places: SentResourceTypes.LINE_LISTING_DATA or something.

Reply:
fixed